contracts/custom/DecentVotesERC20.sol

```diff
@@ -0,0 +1,17 @@
+//SPDX-License-Identifier: MIT
+pragma solidity =0.8.19;
+
+import { VotesERC20 } from "../VotesERC20.sol";
+
+contract DecentVotesERC20 is VotesERC20 {
+
+    error NotDelegatable();
+
+    function delegates(address account) public view override returns (address) {
+        return account;
+    }
+
+    function _delegate(address, address) internal override {
+        revert NotDelegatable();
```

Comment:
internally if a user attempts to delegate, it just reverts.  This impacts only the public delegate() functions of `ERC20VotesUpgradeable`

https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol